src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -77,6 +77,23 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, String filePath) th
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
 
+    @Override
+    public void backupAddressBook(ReadOnlyAddressBook addressBook) throws IOException {
+        String backupAddressBookFilePath = createBackupAddressBookFilePath(addressBookStorage.getAddressBookFilePath());
+        logger.info("Attempting to backup data to data file: " + backupAddressBookFilePath);
+        saveAddressBook(addressBook, backupAddressBookFilePath);
+    }
+
+    /**
+     * Creates file path of the backup data file.
+     * @param addressBookFilePath cannot be null.
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value)?

Reply:
Furthermore if addressBookFilePath cannot be null, your code will not be able to catch such exception when null is passed.

Reply:
@LimYangSheng